upgrade/scripts/k8s/rr_critical_service_restart.py

```diff
@@ -105,7 +105,10 @@ def rollout_restart_critical_services(critical_services: Dict[str, ServiceDetail
             result = subprocess.run(get_command, check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
             resource_json = json.loads(result.stdout)
         except subprocess.CalledProcessError as e:
-            print_stderr(f"Failed to get {resource_type}/{name} in namespace {namespace}: {e.stderr}")
+            if f"Error from server (NotFound): {resource_type.lower()}s.apps \"{name}\" not found" in e.stderr:
+                print(f"Skipping {resource_type}/{name}: resource not found in namespace {namespace}")
+                continue
+            print(f"Failed to get {resource_type}/{name} in namespace {namespace}: {e.stderr}")
```

Comment:
Is there a reason we want this error message going to stdout instead of stderr?
```suggestion
                print(f"Skipping {resource_type}/{name}: resource not found in namespace {namespace}")
                continue
            print_stderr(f"Failed to get {resource_type}/{name} in namespace {namespace}: {e.stderr}")
```